capstone/src/main/webapp/login.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Login</title>
+    <link href="https://fonts.googleapis.com/css2?family=Vollkorn:wght@400;600;800&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+    <script src="https://apis.google.com/js/platform.js" async defer></script>
+    <meta name="google-signin-client_id" content='914921573408-h9cb4rja119k5c92jaa669lvn3q4fvlc.apps.googleusercontent.com'>
```

Comment:
Care for client id

Reply:
Also since you move the google button here, I think it should be good to remove the apis.google.com scriptt and the meta tag from index.html.

Reply:
Got it